include/actions/AddDriftDiffusionAction.h

```diff
@@ -49,5 +54,7 @@ class AddDriftDiffusionAction : public Action
                          const std::string & potential_name,
                          const int & component);
 
+  const bool _use_ad;
+  std::string _ad_prepend;
   // virtual void addEEDFCoefficient(const unsigned & reaction_num);
```

Comment:
Could you remove this old comment while making these other changes?

Reply:
Got it! 👍